src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Smartphones</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#"
+              class="navigation__link"
+              data-qa="nav-hover">
+            Laptops & Computers
+            </a>
```

Comment:
minor - line-wrapping
it is better for readability to have each attribute on a new line. Also each should be indented with 2 spaces:

```suggestion
            <a 
              href="#"
              class="navigation__link"
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
```